_episodes/03-exceptions.md

```diff
@@ -1,29 +1,34 @@
 ---
-layout: page
-title: Testing
-subtitle: Exceptions
-minutes: 10
+title: Exceptions
+teaching: 10
+exercises: 0
+questions:
+- "How do I capture errors that I do not anticipate?"
```

Comment:
It's important that we communicate that these are runtime errors. Anyway, you have to anticipate them to correctly write an exception. I'd phrase this differently. Perhaps "How do I handle unusual behavior while the code runs?"